Frameworks/CoreText/CTLine.mm

```diff
@@ -278,21 +303,24 @@ void CTLineDraw(CTLineRef lineRef, CGContextRef ctx) {
  @Status Interoperable
 */
 CFIndex CTLineGetGlyphCount(CTLineRef line) {
-    return line ? static_cast<_CTLine*>(line)->_glyphCount : 0;
+    RETURN_RESULT_IF_NULL(line, 0);
+    return line->_glyphCount;
```

Comment:
It would be ideal, but the earlier code violated the getter/setter pattern and exposed private vars outside the file.
so it doesn't help us much here.

---
In reply to: [112724807](https://github.com/Microsoft/WinObjC/pull/2586#discussion_r112724807) [](ancestors = 112724807)